client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { localize } from 'i18n-calypso';
+
+class WaitingTwoFactorNotificationApproval extends Component {
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<form onSubmit={ this.onSubmitForm }>
+					<Card className="is-compact">
+						<p>
+							{ translate( 'We just sent a push notification to your WordPress mobile app. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'We just sent a push notification to your <strong>WordPress mobile app</strong>. Swipe or tap to open and verify your login.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).